src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/SelectedCourtService.java

```diff
@@ -61,6 +65,18 @@ public void setSelectedCourtDetailsIfPresent(FinremCaseData caseData) {
                 selectedCourtId);
     }
 
+    /**
+     * Contested Form A creation does not permit these courts to be chosen.
+     * This simple check returns true if caseData shows that a User has picked either of
+     * these courts.
+     *
+     * @param caseData instance of FinremCaseData
+     */
+    public boolean royalCourtOrHighCourtChosen(FinremCaseData caseData) {
+        String courtName = caseData.getConsentOrderWrapper().getConsentOrderFrcName();
+        return courtName != null && Set.of(HIGH_COURT, ROYAL_COURT).contains(courtName.toUpperCase());
```

Comment:
It would make more sense to create this set as a member variable rather than creating a new one each time the method is invoked.
Also perhaps reconsider the name of the method. If you add a new court to the list then you'd presumably have to rename it.

Reply:
Thanks @jthmcts.  I thought on the method name a bit.  Opted to make it nice and plain English.  In this situation the courts are unlikely to change.  If they do, then that's the time of change the method name.